src/Common/Commands.Common/PowerShellUtilities.cs

```diff
@@ -93,5 +93,16 @@ public static IEnumerable<RuntimeDefinedParameter> GetUsedDynamicParameters(Runt
         {
             return dynamicParameters.Values.Where(dp => MyInvocation.BoundParameters.Keys.Any(bp => bp.Equals(dp.Name)));
         }
+
+        public static string FormatErrorRecord(ErrorRecord record)
```

Comment:
Yes, I'm extracting the code from https://github.com/Azure/azure-powershell/pull/3663/files#diff-90463d182d91894d58358adc66b14651L117 as a method. This file seemed like the logical place to put it.